manage.py

```diff
@@ -4,7 +4,7 @@
 
 if __name__ == "__main__":
     if 'test' in sys.argv:
-        os.environ.setdefault("DJANGO_SETTINGS_MODULE", "lti_emailer.settings.unit_test")
+        os.environ.update({"DJANGO_SETTINGS_MODULE": "lti_emailer.settings.unit_test"})
```

Comment:
it's just the one value, why not `os.environ['DJANGO_SETTINGS_MODULE'] = 'lti_emailer.settings.unit_test'`?